public/app/features/datasources/settings/DataSourceSettingsPage.tsx

```diff
@@ -36,10 +37,12 @@ export interface Props {
   setDataSourceName: typeof setDataSourceName;
   updateDataSource: typeof updateDataSource;
   setIsDefault: typeof setIsDefault;
+  plugin?: DataSourcePlugin<any, any>;
```

Comment:
you do not need the any, any the types are optional?

Reply:
You are right, but Options are not optional. Gonna make options default to be {}